src/guides/v2.3/javascript-dev-guide/widgets/widget_calendar.md

```diff
@@ -41,6 +41,28 @@ require([
 });
 ```
 
+## Validate the calendar widget
+
+To validate the calendar widget, required **dateFormat** parameter should be specified, as can be seen from the **validate-date** rule initialisation:
+
+```javascript
+'validate-date': [
+            function (value, params, additionalParams) {
```

Comment:
Please, make sure we use the correct indentation (number of spaces before each line of code) here. 
Let me provide a correct version: 

```javascript
'validate-date': [
    function (value, params, additionalParams) {
        var test = moment(value, utils.convertToMomentFormat(additionalParams.dateFormat));

        return $.mage.isEmptyNoTrim(value) || test.isValid();
    },
    $.mage.__('Please enter a valid date.')
]
```